orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/job/RunJobStage.groovy

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.pipeline.job
+
+import com.netflix.spinnaker.orca.clouddriver.tasks.MonitorKatoTask
+import com.netflix.spinnaker.orca.clouddriver.tasks.job.JobForceCacheRefreshTask
+import com.netflix.spinnaker.orca.clouddriver.tasks.job.RunJobTask
+import com.netflix.spinnaker.orca.pipeline.LinearStage
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import org.springframework.batch.core.Step
+import org.springframework.stereotype.Component
+
+@Component
+class RunJobStage extends LinearStage {
+  public static final String PIPELINE_CONFIG_TYPE = "runJob"
+
+  RunJobStage() {
+    super(PIPELINE_CONFIG_TYPE)
+  }
+
+  @Override
+  List<Step> buildSteps(Stage stage) {
```

Comment:
At first glance, I suspect we could generify the `CreateServerGroupStage` to support this?

Slightly different behavior (_as you called out_) around forceCacheRefresh ... but that doesn't seem unreasonably to allow cloud providers to parameterize?